sdk/translation/ai-translation-text/package.json

```diff
@@ -0,0 +1,117 @@
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"*.{js,json}\"  \"test/**/*.ts\"",
+    "generate:client": "echo skipped",
+    "integration-test:browser": "dev-tool run test:browser",
+    "integration-test:node": "dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts",
+    "pack": "npm pack 2>&1",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
+    "test": "npm run clean && npm run build:test && npm run unit-test",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "unit-test:node": "dev-tool run test:node-ts-input -- --timeout 1200000 --exclude 'test/**/browser/*.spec.ts' 'test/**/*.spec.ts'",
+    "unit-test:browser": "dev-tool run test:browser",
+    "build": "npm run clean && tsc -p . && dev-tool run bundle && mkdirp ./review && api-extractor run --local"
+  },
+  "sideEffects": false,
+  "autoPublish": false,
+  "dependencies": {
+    "@azure/core-auth": "^1.3.0",
+    "@azure-rest/core-client": "^1.1.0",
+    "@azure/core-rest-pipeline": "^1.8.0",
+    "@azure/logger": "^1.0.0",
+    "tslib": "^2.2.0"
+  },
+  "devDependencies": {
+    "@microsoft/api-extractor": "^7.31.1",
+    "autorest": "latest",
+    "@types/node": "^14.0.0",
+    "dotenv": "^16.0.0",
+    "eslint": "^8.0.0",
+    "mkdirp": "^2.1.2",
+    "prettier": "^2.5.1",
+    "rimraf": "^3.0.0",
+    "source-map-support": "^0.5.9",
+    "typescript": "~4.8.0",
+    "@azure/dev-tool": "^1.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@azure-tools/test-credential": "^1.0.0",
+    "@azure/identity": "^2.0.1",
+    "@azure-tools/test-recorder": "^3.0.0",
+    "mocha": "^7.1.1",
+    "@types/mocha": "^7.0.2",
+    "mocha-junit-reporter": "^1.18.0",
+    "cross-env": "^7.0.2",
+    "@types/chai": "^4.2.8",
+    "chai": "^4.2.0",
+    "karma-chrome-launcher": "^3.0.0",
+    "karma-coverage": "^2.0.0",
+    "karma-env-preprocessor": "^0.1.1",
+    "karma-firefox-launcher": "^1.1.0",
+    "karma-junit-reporter": "^2.0.1",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-mocha": "^2.0.1",
+    "karma-source-map-support": "~1.4.0",
+    "karma-sourcemap-loader": "^0.3.8",
+    "karma": "^6.2.0",
+    "nyc": "^15.0.0"
+  },
+  "//metadata": {
+    "constantPaths": [
+      {
+        "path": "src/textTranslation.ts",
+        "prefix": "userAgentInfo"
+      }
+    ]
+  },
+  "browser": {
+    "./dist-esm/test/public/utils/env.js": "./dist-esm/test/public/utils/env.browser.js"
+  }
```

Comment:
Looks like this module doesn't exist?
```suggestion
```

Reply:
It is showing that you want to delete just line 126 on my end, do you want to remove whole: 
```json
  "browser": {
    "./dist-esm/test/public/utils/env.js": "./dist-esm/test/public/utils/env.browser.js"
  }
```

<img width="575" alt="image" src="https://user-images.githubusercontent.com/5716362/230242941-7217640b-331a-42d1-82dc-7a137d2bbc4c.png">

Reply:
Yes, L124-126 inclusive

Reply:
Deleted